src/helpers/filterTodos.ts

```diff
@@ -0,0 +1,24 @@
+import { Todo } from '../types/Todo';
+
+import { FilterType } from '../enums/FilterType';
+
+export function filterTodos(
+  todos: Todo[],
+  selectedFilter: FilterType,
+): Todo[] {
+  return todos.filter((todo) => {
+    switch (selectedFilter) {
+      case FilterType.All:
+        return true;
```

Comment:
it's a useless iteration. You should do different filters inside every case block